src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -36,20 +36,20 @@ With [our Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricin
+  The <DoNotTranslate>**Data retention**</DoNotTranslate> UI is where you can see your data retention periods, upgrade your retention plan (Max per contract) and edit effective retention. To view explanations of the table column headers, hover over a header. 
 </figcaption>
 
 ## Requirements [#requirements]
 
-Being able to configure data retention requires Pro or Enterprise edition with billing administrator rights. Account characteristics that do not meet this criteria will block some New Relic organizations from adjusting retention on their own. If you're a Pro or Enterprise organization on a billing administrator account, and you can't edit retention, talk to your New Relic account representative.
+Being able to upgrade your data retention plan ((Max per contract) or edit the effective retention values requires Pro or Enterprise. Account characteristics that do not meet this criteria will block some New Relic organizations from adjusting upgrading and editing effective retention values on their own. If you're a Pro or Enterprise and you can't upgrade your retention plan or edit retentionv values, talk to your New Relic account representative.
 
 Note that our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus) provides up to 
 an additional 90 days of retention.
 
 Permission-related requirements for viewing and editing retention:
 * [Newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models): 
-  * Setting a contract with longer data retention for your entire organization requires the [<DoNotTranslate>**Billing**</DoNotTranslate> administration setting](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings)
-  * Adjusting retention within the bounds of a contract for specific types of data or for a specific account requires the [<DoNotTranslate>**Insights event retention**</DoNotTranslate> permission](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#data-retention) 
+  * Setting a contract with longer data retention allowance for your entire organization (Max per contract) requires the [<DoNotTranslate>**Billing**</DoNotTranslate> administration setting](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings)
+  * Adjusting effective retention within the bounds of a contract ((Max per contract) for specific types of data or for a specific account requires a custom role with View and Modify for the [<DoNotTranslate>**Insights event retention**</DoNotTranslate> capability](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#data-retention)  access grant pointed at the account(s) the user wants to be able to edit the effective retention values for.
```

Comment:
```suggestion
  * You need to have a custom role to adjust the effective retention within the bounds of a contract (max per contract) for specific types of data or for a specific account. The custom role needs to have the **View** and **Modify** permissions for the [<DoNotTranslate>**Insights event retention**</DoNotTranslate> capability](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#data-retention). Also, the role needs to be in a group that applies to the account(s) where the user wants to edit the effective retention.
```

Reply:
Unfortunately you took out something that is separate from being able to edit the effective retention (which is what you reduced it down to). We still need to cover what is needed to upgrade your data retention (aka increase your max per contract values) which is what the first blurb I had edited was speaking to.

Reply:
Oh wait, I see you only edited the second blurb? If so, yours does look better :)

Reply:
@johnathannewrelic I'm seeing two bullets here: the first is about setting a longer retention through **Max per contract** (no changes) and the second bullet is about changing the effective retention.

Are we saying that the custom role is only relevant for changing effective retention? If that's the case my revision might cut it. If the custom role is required for both setting longer retention and effective retention, we'll need another revision, I think.

In this second bullet about effective retention, why is there also a reference to **Max per contract**? I had thought that column only applied to setting longer retention. Are you saying that effective retention is set within the upper boundary of **Max per contract**?

Reply:
The first is about upgrading/increasing your **_Max per contract_** values which is generally determined by your subscription choices (for example Data Plus), Your *Max per contract determines how high you can set effective retention values to for the various types of data ingested, upgrading your Max per contract requires Billing role for the billing/reporting account ID of the organization. When you click on the Upgrade retention button (which again is only visible to a user with Billing role) it essentially brings you to a plan upgrade page.

The second is about setting _**effective retention**_ for a specific type of data for an account in your org, which you can only set to a value that is within your Max per contract value and doing this requires Insight Events Retention capability which is a capability that _can_ be given via Billing role, but only for the billing/reporting account ID of the organization which is not a problem if you only have one account in your organization. If you have more than one account in your organization and you want to be able to change your effective retention values for an account other than your billing/reporting ID account, then you would need to create a custom role that has View and Modify for the Insight Events Retention capability and point it to those accounts.